templates/zerver/help/import-users-and-channels-from-slack.md

```diff
@@ -0,0 +1,45 @@
+# Import users and channels from Slack
+
+{!follow-steps.md!} import users and channels from Slack to Zulip.
+
+
+!!! warn ""
+    **Note:** Please ensure the user has admin rights.
```

Comment:
"**Note:** Please ensure that you have administrator rights before importing users and channels from Slack" ("the user" is vague here since it's third person, someone reading this might think it doesn't refer to themselves but someone else)